web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
Hi Sylvain, 
If subject is to avoid one click to see what are the tables.models, then promote to remove 'models' folder, and then you directly jump into the file with the whole code. 
In small modules (80 % of my contributions) they only have 1-2 files in model folder.
Ok, I go out now ;-)

PS: you can split my file in part if you want, no offense for me